src/content/api/cli.md

```diff
@@ -44,13 +45,45 @@ webpack-cli offers a variety of commands to make working with webpack easy. By d
 | `configtest` | `configtest\|t [config-path]`             | Validate a webpack configuration.                                               |
 | `help`       | `help\|h [command] [option]`              | Display help for commands and options.                                          |
 | `info`       | `info\|i [options]`                       | Outputs information about your system.                                          |
-| `init`       | `init\|c [scaffold...] [options]`         | Initialize a new webpack configuration.                                         |
+| `init`       | `init\|c [generation-path] [options]`     | Initialize a new webpack project.                                               |
 | `loader`     | `loader\|l [output-path]`                 | Scaffold a loader.                                                              |
 | `plugin`     | `plugin\|p [output-path]`                 | Scaffold a plugin.                                                              |
 | `serve`      | `serve\|s [options]`                      | Run the `webpack-dev-server`.                                                   |
 | `version`    | `version\|v [commands...]`                | Output the version number of `webpack`, `webpack-cli` and `webpack-dev-server`. |
 | `watch`      | `watch\|w [entries...] [options]`         | Run webpack and watch for files changes.                                        |
 
+### Init
+
+Used to initialize a new webpack project.
+
+```bash
+npx webpack init [generation-path] [options]
+```
+
+**example**
+
+```bash
+npx webpack init ./my-app --force --template=default
```

Comment:
as `--force` will use the default.

```suggestion
npx webpack init ./my-app --force
```

Reply:
I want to explain all possible way to use command in a single example, which includes specifying template.